scripts/apps/authoring/media/MediaMetadataEditorDirective.ts

```diff
@@ -1,10 +1,16 @@
 import {max, sortBy, get} from 'lodash';
 import {getLabelNameResolver} from 'apps/workspace/helpers/getLabelForFieldId';
 
-MediaMetadataEditorDirective.$inject = ['metadata', 'deployConfig', '$q'];
-export default function MediaMetadataEditorDirective(metadata, deployConfig, $q) {
+MediaMetadataEditorDirective.$inject = ['metadata', 'deployConfig', 'features', '$q'];
+export default function MediaMetadataEditorDirective(metadata, deployConfig, features, $q) {
     function getCV(field) {
-        return metadata.cvs.find((cv) => !cv.field_type && (cv._id === field || cv.schema_field === field));
+        const cv = metadata.cvs.find((cv) => !cv.field_type && (cv._id === field || cv.schema_field === field));
+
+        if (cv == null && field === 'subject') {
```

Comment:
It'd be useful to leave a comment here about what's going on. Why is the custom vocabulary exceptional instead of being included in `metadata.cvs`?